engine/src/main/java/org/destinationsol/game/drawables/animated/AnimatedRectSprite.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.game.drawables.animated;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.Animation;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.math.Vector2;
+import org.destinationsol.common.Consumed;
+import org.destinationsol.game.SolGame;
+import org.destinationsol.game.SolObject;
+import org.destinationsol.game.drawables.DrawableLevel;
+import org.destinationsol.game.drawables.RectSprite;
+
+//TODO: Should this allow variable-size frames?
```

Comment:
No. There is no real reason to support that, since almost every single engine only has support for fixed sized frames, and it certainly poses no limitation.